### ISSUE TITLE
Have ThemedButton always use aria-disabled over disabled

### ISSUE THREAD

**Issue by gsproston-scottlogic** [ScottLogic/prompt-injection#598]
# Feature Request

## Description
Currently, the ThemedButton component has both `disabled` and `ariaDisabled` props. Really we should only use `aria-disabled` as it's more accessible. 

## Additional context
Using `aria-disabled` rather than `disabled` on HTML button elements is more accessible as detailed [here](https://css-tricks.com/making-disabled-buttons-more-inclusive/).

Some advantages are:

- Buttons with `aria-disabled` can still be focused, and therefore read out by screen readers.
- Once focused/hovered, we can show a tooltip explaining why the button is not enabled.

At the time of writing this, this issue will affect the following:

- Send button, which is disabled while sending a message.
- Level selection buttons, which are disabled if the user has not unlocked that level yet.

Also, as part of this issue, remove the `appearsDifferentWhenDisabled` prop from the ThemedButton component. 
Instead, just overwrite styling in the components that use it. e.g. LoadingButton.

## Acceptance criteria
**GIVEN** the user has only unlocked level 1
**WHEN** the user changes focus by tabbing through the page
**THEN** the user is able to focus on the buttons for the unlocked levels

**GIVEN** the user has only unlocked level 1
**WHEN** the user focuses on a button for a level that's not yet unlocked
**THEN** a tooltip is displayed explaining why the button is disabled
**AND** the screen reader reads out the name of the button 
**AND** the screen reader reads out that the button is disabled

**GIVEN** the user sends a chat message
**WHEN** the user changes focus by tabbing through the page
**THEN** the user is able to focus on the Send loading button

**GIVEN** the user sends a chat message
**WHEN** the user focuses on the Send loading button
**THEN** a tooltip is displayed explaining why the button is disabled
**AND** the screen reader reads out the name of the button 
**AND** the screen reader reads out that the button is disabled

**Comment by pmarsh-scottlogic** [comment]
TESTING:

Can tab onto the disabled level buttons
**No tooltips!**
cannot click the disabled level buttons
screen reader says "level 1 unavailable", "level 2 unavailable" etc.

Can tab onto the send button while a message is sending
**No tooltip**
cannot click the sending message button
screen reader says "send button unavailable"

Bringing back to review because we are missing the tooltips from the AC

**Comment by gsproston-scottlogic** [comment]
Reopening as the tooltips are not behaving when focusing on the disabled button.
Either they're too far away from the original button (in Edge), or not present at all (in Firefox).
MDN actually recommends AGAINST the use of the `title` attribute for tooltips, and links to [this article](https://inclusive-components.design/tooltips-toggletips/). I'll give that a read and come up with a better solution.
